demosplan/DemosPlanAssessmentTableBundle/Logic/AssessmentTableServiceStorage.php

```diff
@@ -115,6 +116,7 @@ class AssessmentTableServiceStorage
      * @var CurrentUserService
      */
     private $currentUser;
+    private StatementDeleter $statementDeleter;
```

Comment:
Using `private readonly StatementDeleter $statementDeleter` in the constructor would be more concise.

Reply:
a6b381f0b78add7ef5b84ba014ad19745b646ac0